### DISCUSSION TITLE
Using ifcopenshell within IES VE

### DISCUSSION THREAD

**Discussion by JosefUIBK** [IfcOpenShell/IfcOpenShell#5519] [General]
Hello,

i really would like to use the ifcopenshell python framework within the IES VE VEScript Pathon Editor, but i just can't get it to run. Maybe Someone can help.

Thank you

**Comment by aothms** [comment]
IfcOpenShell is a C/C++ module and as such only works with CPython implementations. Is it possible that the software you use uses e.g IronPython? In that way there is no easy integration possible. I'm personally not familiar with the software you mention. What errors to you encounter?
